builds/e2e/longhaul.yaml

```diff
@@ -337,4 +339,66 @@ jobs:
           snitch.storage.account: '$(snitch.storage.account)'
           snitch.storage.masterKey: '$(StorageAccountMasterKeyStress)'
           loadGen.message.frequency: '$(loadGen.message.frequency.arm64v8)'
-          longHaul.protocolHead: 'mqtt'
\ No newline at end of file
+          longHaul.protocolHead: 'mqtt'
+          
+################################################################################
+  - job: windows_amd64_moby
+################################################################################
+    displayName: Windows AMD64 Moby
+    condition: eq(variables['run.windows.amd64.moby'], 'true')
+    pool:
+      name: $(pool.name)
+      demands:
+        - run-long-haul -equals true
+        - Agent.OS -equals Windows_NT
+        - Agent.OSArchitecture -equals X64
```

Comment:
I found that long haul is tested to run on iotedge-win5; I remember I created this VM for Chandler for build edgelet for ARM.  Does Chandler aware of it?

Reply:
Slavi asked Chandler and he is not using that VM. We are also the ones that created those VMs last sprint.